cpp/include/Ice/SSL/ServerAuthenticationOptions.h

```diff
@@ -248,11 +248,11 @@ namespace Ice::SSL
         /// the connection will be aborted with an Ice::SecurityException.
         ///
         /// @param verified A boolean indicating whether the preliminary certificate verification performed by OpenSSL's
-        /// built-in mechanisms succeeded or failed. True if the preliminary checks passed, false otherwise.
+        /// built-in mechanisms succeeded or failed. True if the preliminary checks passed, `false` otherwise.
```

Comment:
I assume this should be changed as well?
```suggestion
        /// built-in mechanisms succeeded or failed. `true` if the preliminary checks passed, `false` otherwise.
```